lib/api/apiUtils/coldStorage.js

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
I realized that our `coldStorage.js` files are not under the same folder, mine is under `lib/api/apiUtils/object/coldStorage.js`, which one do you think makes more sense?

Reply:
I chose to use the same location as yours (and Yutaka) : https://github.com/scality/cloudserver/pull/4504/commits/acef14d85ebf63b394b7a0bfb1c37f83428e9b50
I also rebased the other PR.